covalent/_workflow/lattice.py

```diff
@@ -287,90 +285,13 @@ def check_consumable(self) -> None:
                     )
                 )
 
-    def dispatch(self, *args, **kwargs) -> str:
```

Comment:
@kessler-frost I forgot to add this, can you  have a pass through of this calling `ct.dispatch` with a depreciation warning that this will be removed in few releases?

Reply:
or a meaning full error with a pass passthrough without dispatch.

Reply:
Hmm, why can't we remove it right now? We will be changing the minor patch version indicating that a breaking change has been introduced. Plus I have changed it in the tutorials and how tos so new users won't be affected and old users won't be affected since they don't have this change. Plus, even if I change it in this PR, the only users that are going to see it are those who have installed the latest version. 🤔

Reply:
After some discussion we decided to put the deprecation warning.